sycl/source/detail/accessor_impl.cpp

```diff
@@ -14,8 +14,9 @@ namespace sycl {
 namespace detail {
 
 AccessorImplHost::~AccessorImplHost() {
-  if (MBlockedCmd)
+  if (MBlockedCmd) {
```

Comment:
AFAIK, it's common practice in LLVM not to wrap single line statement in curly brackets.